packages/flutter/lib/src/material/search_anchor.dart

```diff
@@ -307,6 +313,22 @@ class SearchAnchor extends StatefulWidget {
   /// ```
   final BoxConstraints? viewConstraints;
 
+  /// The padding to use for the search view.
+  ///
+  /// Has no effect if the search view is full-screen.
+  ///
+  /// If null, the value of [SearchViewThemeData.padding] will be used.
+  /// If this is also null, then the padding defaults to `EdgeInsets.only(bottom: 16)`.
```

Comment:
Actually I think we don't really need a default because I don't see this from Material specs. We can just keep this nullable and remove the default value so we won't cause any breaking changes to existing apps.